.travis.yml

```diff
@@ -1,5 +1,8 @@
 ---
+notifications:
+  email: false
 language: ruby
+before_install: rm Gemfile.lock || true
```

Comment:
There's no `Gemfile.lock` in the repository so this line shouldn't be necessary.